.eslintrc.json

```diff
@@ -11,7 +11,10 @@
   "rules": {
     "indent": [
       "error",
-      2
+      2,
+      {
+        "SwitchCase": 1
+      }
```

Comment:
This makes `switch` statements look like 
```js
switch (x) {
  case 42:
    console.log("BANANA!");
}
```
instead of 
```js
switch (x) {
case 42:
  console.log("BANANA!");
}
```